bin/run-nightly-functional-tests.sh

```diff
@@ -1,21 +1,7 @@
 #!/bin/bash
 set -ex
 
+TEST_URL='https://probate.aat.platform.hmcts.net/caveats/start-apply'
```

Comment:
Is this needed if the URL is passed in by the Jenkins_nightly file? by hardcoding it here, it defeats the purpose of making it manually configurable

Reply:
let's have call regarding this?

Reply:
Jenkins nightly pipeline passing https://probate.aat.platform.hmcts.net/' and it's not passing above URL